### PR TITLE
support for multilib build of GCC on PowerPC

### PR DESCRIPTION
**Pull request by JackPerdue**

### CODE REVIEW

File: easybuild/easyblocks/g/gcc.py [hunk 1]
```diff
@@ -249,6 +249,7 @@ def configure_step(self):
         if self.cfg['multilib']:
             glibc_32bit = [
                 "glibc.i686",  # Fedora, RedHat-based
+                "glibc.ppc",   # "" on Power
```
Comment (boegel) [inline easybuild/easyblocks/g/gcc.py]: @JackPerdue: I'm not sure what you mean with the `""` in the comment?

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyblocks/g/gcc.py (hunk 1 above)
Comment (JackPerdue) [inline easybuild/easyblocks/g/gcc.py]: "" == 'same as above'... i.e. Fedora, Redhat-based

Comment (boegel) [inline easybuild/easyblocks/g/gcc.py]: OK, makes sense.

**Comment by boegel** [comment]
@wpoely86: do you see any reason not to merge this?

Should we add a check to only consider the `.ppc` one when we're on a Linux/POWER system, and only consider the ones we have now when we're on a Linux/x86_64 system?

**Comment by wpoely86** [comment]
@boegel that's probably a bit far fetched. If you have glibc.ppc on a x86_64, you're doing weird stuff. 

Good to merge IMHO

**Comment by JackPerdue** [comment]
Alas, I don't know the equivalent for Debian or SUSE (or have a way to test) so that might be good later.

**Comment by boegel** [comment]
Merging this one, thanks @JackPerdue!

**Pull request merged**